public/scripts/dashboard.js

```diff
@@ -25,6 +25,7 @@ function setDataIntoDashboard() {
 
             displayDataIntoDashboard(totalDevelopmentTask, totalBusinessTask, totalMarketingTask, totalCompletedTask, startDate, endDate, status);
             getTasks();
+            getBacklog();
```

Comment:
Similar method getTasks is already doing this.

Reply:
done